#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PKG = "jsk_topic_tools"

gen = ParameterGenerator()

#       name    type     level     description     default      min      max
gen.add("queue_size", int_t, 0, "Queue size of subscriber", 1, 1, 1000)
gen.add("enable_monitor", bool_t, 0, "Enable monitoring", True)
gen.add("monitor_topic", str_t, 0, "Name of monitoring topic", "")
gen.add("monitor_rate", double_t, 0, "Desired monitoring rate of subscribing messages", 1.0, 0.001, 60.0)

exit(gen.generate(PKG, PKG, "StealthRelay"))
